app/models/organization_user.rb

```diff
@@ -1,4 +1,8 @@
 class OrganizationUser < ActiveRecord::Base
   belongs_to :organization
   belongs_to :user
+
+  def invite_mail(invited_by)
```

Comment:
hmmm, was just about to make this change, and I realized... it's not actually mailing the invite... it's returning an action mailer object.... so `mail_invite` is a bit misleading. I'll go with `invite_mailer` instead, which I think is probably a better alternative.